src/app/core/entity-components/entity-details/related-entities/related-entities.component.ts

```diff
@@ -30,7 +33,15 @@ export class RelatedEntitiesComponent<E extends Entity> implements OnInit {
    */
   @Input() property: string;
 
-  @Input() columns: ColumnConfig[];
+  @Input()
+  public set columns(value: ColumnConfig[]) {
+    this._columns = value;
+  }
+  public get columns(): ColumnConfig[] {
+    return this._columns;
+  }
+  protected _columns: ColumnConfig[];
```

Comment:
Whats the benefit of the getters and setters?

Reply:
These are required here because the other component extending this is working with the setter to add another column on the fly (and to override the base class has to implement it already).